ffv1.md

```diff
@@ -111,6 +111,8 @@ log2(a)               the base-two logarithm of a
 
 min(a,b)              the smallest of two values a and b
 
+median(a,b,c)         the numerical middle value in a data set of a, b, and c
```

Comment:
You can define median() as a+b+c-min(a,b,c)-max(a,b,c) if you want a more formal definiton without "middle"